Casks/d/db-browser-for-sqlcipher@nightly.rb

```diff
@@ -1,12 +1,17 @@
 cask "db-browser-for-sqlcipher@nightly" do
-  version :latest
-  sha256 :no_check
+  version "20250608"
+  sha256 "78a58c22eb75fe0d3aba91b341670669117bcf2d850bd8cce5906873b7a5bbdb"
 
-  url "https://nightlies.sqlitebrowser.org/latest/DB.Browser.for.SQLCipher-universal.dmg"
+  url "https://github.com/sqlitebrowser/sqlitebrowser/releases/download/nightly/DB.Browser.for.SQLCipher-universal_#{version}.dmg",
+      verified: "github.com/sqlitebrowser/sqlitebrowser/"
   name "DB Browser for SQLCipher Nightly"
   desc "Database browser for SQLCipher"
   homepage "https://sqlitebrowser.org/"
 
+  livecheck do
+    cask "db-browser-for-sqlite@nightly"
```

Comment:
```suggestion
    cask "db-browser-for-sqlite@nightly"
    regex(/^DB[._-]Browser[._-]for[._-]SQLCipher[._-]universal[._-]v?(\d+(?:\.\d+)*)\.dmg/i)
```

You can override inherited values when using a `formula`/`cask` reference, so we can use `regex` to set up the specific regex for this cask (instead of using a generic regex for both casks).

Reply:
Neat; I'll mention that in the docs.